<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>273</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Entry Country Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Capital</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="capitalEdit">
       <property name="placeholderText">
        <string>Entry Capital Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Population</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="populationEdit">
       <property name="placeholderText">
        <string>Enter Population</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showInfoButton">
       <property name="text">
        <string>Show Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>Results...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>